modules/extensions/k8stools.tf

```diff
@@ -23,14 +23,23 @@ resource "null_resource" "install_k8stools_on_operator" {
     content     = local.install_helm_template
     destination = "/home/opc/install_helm.sh"
   }
+
+  provisioner "file" {
+    content     = local.install_kubectx_template
+    destination = "/home/opc/install_kubectx.sh"
```

Comment:
You're hard-coding a path here, but below you use `$HOME/install_kubectx.sh` and I see you have a hard-coded path for `install_helm.sh` above. I would recommend using `$HOME` throughout so that you're not tied to the specific username of the generic user.

Reply:
$HOME gets rendered as a string and the file is not written.